<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>985</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>841</width>
      <height>461</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Main">
     <attribute name="title">
      <string>主机信息</string>
     </attribute>
     <widget class="QLabel" name="host_name">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>20</y>
        <width>250</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>主机名</string>
      </property>
     </widget>
     <widget class="QLabel" name="start_time">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>120</y>
        <width>250</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>启动时间</string>
      </property>
     </widget>
     <widget class="QLabel" name="run_time">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>220</y>
        <width>250</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="text">
       <string>运行时间</string>
      </property>
     </widget>
     <widget class="QLabel" name="sys_version_2">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>20</y>
        <width>250</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>系统版本号：</string>
      </property>
     </widget>
     <widget class="QLabel" name="CPU_type">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>320</y>
        <width>250</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>CPU型号：</string>
      </property>
     </widget>
     <widget class="QLabel" name="start_time_2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>120</y>
        <width>250</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>启动时间：</string>
      </property>
     </widget>
     <widget class="QLabel" name="host_name_2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>20</y>
        <width>250</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>主机名：</string>
      </property>
     </widget>
     <widget class="QLabel" name="run_time_2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>220</y>
        <width>250</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="text">
       <string>运行时间：</string>
      </property>
     </widget>
     <widget class="QLabel" name="sys_version">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>20</y>
        <width>250</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>系统版本号</string>
      </property>
     </widget>
     <widget class="QLabel" name="CPU_name">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>320</y>
        <width>411</width>
        <height>78</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>CPU型号：</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Memory">
     <attribute name="title">
      <string>内存资源</string>
     </attribute>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>180</width>
        <height>70</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>内存使用率</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar_mem">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>601</width>
        <height>41</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QLabel" name="mem_line">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>100</y>
        <width>720</width>
        <height>300</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>92</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>100%</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>240</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>50%</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>390</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0%</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="CPU">
     <attribute name="title">
      <string>CPU信息</string>
     </attribute>
     <widget class="QLabel" name="cpu_rate_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>0</y>
        <width>180</width>
        <height>70</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>CPU使用率：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>92</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>100%</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>162</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>75%</string>
      </property>
     </widget>
     <widget class="QLabel" name="cpu_line">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>100</y>
        <width>720</width>
        <height>300</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>240</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>50%</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>310</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>25%</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>390</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>0%</string>
      </property>
     </widget>
     <widget class="QLabel" name="cpu_rate_3">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>0</y>
        <width>180</width>
        <height>70</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>%</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar_cpu">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>541</width>
        <height>41</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="prograss">
     <attribute name="title">
      <string>进程信息</string>
     </attribute>
     <widget class="QListWidget" name="listWidget_process">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>781</width>
        <height>351</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>390</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>刷新</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>390</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>结束</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>500</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Umpush</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>关机</string>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="now_time">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>510</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>now_time</string>
    </property>
   </widget>
   <widget class="QLabel" name="now_time_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>510</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>当前时间：</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
